pkg/admission/clusterworkspacetypeexists/admission.go

```diff
@@ -165,6 +170,52 @@ func (o *clusterWorkspaceTypeExists) Admit(ctx context.Context, a admission.Attr
 	return updateUnstructured(u, cw)
 }
 
+func (o *clusterWorkspaceTypeExists) resolveValidClusterWorkspaceType(parentClusterName logicalcluster.Name, ref tenancyv1alpha1.ClusterWorkspaceTypeReference) (*tenancyv1alpha1.ClusterWorkspaceType, error) {
```

Comment:
everything is about CWTs in here. Just call is `resolveType`. Much easier to read.

Reply:
unresolved